### PR TITLE
Replace config root by config mapping

### PR DESCRIPTION
**Pull request by fjtirado**

### CODE REVIEW

File: quarkus-asyncapi/deployment/pom.xml
Comment (ChMThiel) [inline quarkus-asyncapi/deployment/pom.xml:1]: @fjtirado, should we switch to ConfigMapping as described here https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.14#for-extension-developers? Are there any problems on your side or was is just the quick&easy solution to enable the legacy-property?

Comment (ChMThiel) [reply quarkus-asyncapi/deployment/pom.xml:1]: After some reading, i understand, that 'per default' all config-properties have to start with 'quarkus' - a breaking change. 
There is a workaround https://github.com/orgs/quarkiverse/discussions/228#discussioncomment-10455441 implementing a  smallrye config customizer... i will give that a try...

Comment (fjtirado) [reply quarkus-asyncapi/deployment/pom.xml:1]: @ChMThiel  To be honest, I took the simplest approach, which was to specify the legacy property. Let me know if you try to change to ConfigMapping

Comment (ChMThiel) [reply quarkus-asyncapi/deployment/pom.xml:1]: > @ChMThiel To be honest, I took the simplest approach, which was to specify the legacy property. Let me know if you try to change to ConfigMapping

I already did: see https://github.com/quarkiverse/quarkus-asyncapi/commit/a7bdd4d13f6c8984390bde0f4f3d2a5d04411591

**Pull request merged**